.editorconfig

```diff
@@ -11,3 +11,6 @@ ktlint_standard_property-naming = disabled
 # `string-template-indent` is disabled because it depends on `multiline-expression-wrapping`.
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+
+[sample/**/*.{kt,kts}]
+ktlint_standard_no-multi-spaces = disabled
```

Comment:
Disabled this rule for the `sample` module because we commonly use lots of spaces for numbering steps, for example:

https://github.com/JuulLabs/krayon/blob/8ec24878d3df010aec28ece5f24471905b0a7144/sample/src/jsMain/kotlin/tutorial/Line1.kt#L13-L22